build.sbt

```diff
@@ -59,15 +59,12 @@ lazy val scoverageSettings: Seq[Setting[_]] = Seq(
   coverageExcludedPackages := List(
     "<empty>"
     ,"Reverse.*"
-    ,"models\\..*"
-    ,"forms\\..*"
-    ,"views\\..*"
-    ,"domain\\..*"
     ,"metrics\\..*"
     ,"features\\..*"
+    ,"test\\..*"
     ,".*(BuildInfo|Routes|Options).*"
   ).mkString(";"),
-  coverageMinimum := 70,
+  coverageMinimum := 83,
```

Comment:
👍  Did you choose an actual value? :D

Reply:
just bumped it up to what it was passing at so people are encouraged to not let this keep slipping down to 70%